apis/v1alpha1/instrumentation_types.go

```diff
@@ -97,8 +97,36 @@ type Resource struct {
 // Exporter defines OTLP exporter configuration.
 type Exporter struct {
 	// Endpoint is address of the collector with OTLP endpoint.
+	// The TLS is enabled
```

Comment:
or should we say if the endpoint contains `https` a `tls` block must be specified?

Reply:
certainly, in the future it would be great if we could just specify a collector name and namespace instead of an arbitrary string, do some validation and then create the certs for both.

Reply:
>  certainly, in the future it would be great if we could just specify a collector name and namespace instead of an arbitrary string, do some validation and then create the certs for both. 

We should probably offer both. Otherwise it would no longer be possible to report to a custom service or a self-deployed collector?

Reply:
more direct integration with collector will be also needed if we want to support auto-provisioning of certs.